neo4j/assets/service_checks.json

```diff
@@ -1,17 +1 @@
-[
-    {
-        "agent_version": "6.3.0",
-        "integration": "Neo4j",
-        "check": "neo4j.can_connect",
-        "statuses": [
-            "ok",
-            "critical"
-        ],
-        "groups": [
-            "server_name",
-            "url"
-        ],
-        "name": "Can Connect",
-        "description": "Returns `CRITICAL` if the Agent fails to receive a 200 status from the monitoring endpoint, otherwise returns `OK`."
-    }
-]
+[]
```

Comment:
Why are you removing this file?

Reply:
Newest version has no service checks. Is that a requirement?